src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -1225,46 +1265,69 @@ namespace Google.Example.Library.V1.Samples
     using System;
-    using System.Threading.Tasks;
+    using System.Collections.Generic;
 
-    public class GetBookAsyncFlattenedAsyncTestOnSuccessMap
+    public class GetBigBookAsyncLongRunningFlattenedAsyncPollUntilCompleteWap
     {
         /// <summary>
-        /// When there is a very long description for a sample file we can split
-        /// the description into multiple lines and the samples will be just fine
+        /// Testing calling forms
         /// </summary>
-        public static async Task SampleGetBookAsync()
+        public static async Task SampleGetBigBookAsync(string shelf)
         {
             LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
-            BookName name = new BookName("[SHELF_ID]", "[BOOK_ID]");
-            Book response = await libraryServiceClient.GetBookAsync(name);
-            string intKeyVal = response.MapStringValue[123];
-            string booleanKeyVal = response.MapBoolKey[true];
-            int mapValueField = response.MapMessageValue["key"].Field;
-            Console.WriteLine($"Test printing map fields: {response.MapListValueValue["quoted_key"]}");
-            Console.WriteLine($"Test printing enum fields of a map value: {response.MapMessageValue["key"].Alignment}");
+            // string shelf = "Novel\\\"`\b\t\n\r"
+            BookName name = new BookName(shelf, "War and Peace");
+            // Poll until the returned long-running operation is complete
+            Operation<Book, GetBigBookMetadata> operation = await libraryServiceClient.GetBigBookAsync(name);
+            Book response = (await operation.PollUntilCompletedAsync()).Result;
+            // Testing iterating over map fields when both key and value are specified.
+            foreach (KeyValuePair<string, ListValue> entry in response.MapListValueValue.entrySet()) {
+                string myKey = entry.Key;
+                ListValue myValue = entry.Value;
+                Console.WriteLine($"key: {myKey}, value: {myValue}");
+            }
+            // Testing iterating over map fields when only key is specified.
+            foreach (var anotherKey in response.MapListValueValue.Keys) {
+                Console.WriteLine($"key: {anotherKey}");
+            }
+            // Testing iterating over map fields when only value is specified.
+            for (var anotherValue in response.MapListValueValue.Values) {
+                Console.WriteLine($"value: {anotherValue}");
+            }
+            Console.WriteLine($"name: {response.Name}");
+            Console.WriteLine($"author: {response.Author}");
         }
-        // [END sample]
+        // [END hopper]
         public static void Main(string[] args)
         {
-            SampleGetBookAsync().Wait();
+            new Parser(with => with.CaseInsensitiveEnumValues = true).ParseArguments<Options>(args)
+                .WithParsed<Options>(opts =>
+                    SampleGetBigBook(opts.Shelf));
```

Comment:
This should be calling the async method, not the sync method (which doesn't exist).

Reply:
Done